content/patterns/disclosure/examples/disclosure-faq.html

```diff
@@ -18,146 +18,147 @@
+          <li>視覚障害のある人がディスクロージャーをインタラクティブなものとして識別し、ディスクロージャーボタン又はそのラベルをクリックすると展開状態が変化することを認識しやすくするために、ポインターがボタン又はそのラベルにホバーすると、背景色が変わります。境界線が表示され、カーソルがポインターに変わります。</li>
           <li>
-            Because transparent borders are visible on some systems with operating system high contrast settings enabled, transparency cannot be used to create a visual difference between the element that is focused and other elements.
-            Instead of using transparency, the focused element has a thicker border and less padding.
-            When an element receives focus, its border changes from 0 to 2 pixels and padding is reduced by 2 pixels.
-            When an element loses focus, its border changes from 2 pixels to 0 and padding is increased by 2 pixels.
+            透明な境界線は、オペレーティングシステムのハイコントラスト設定が有効になっている一部のシステムで表示されるため、フォーカスされている要素と他の要素との間に視覚的な違いを作成するために透明度を使用することはできません。
+            透明度を使用する代わりに、フォーカスされた要素はより太い境界線とより少ないパディングを持ちます。
+            要素がフォーカスを受け取ると、その境界線は 0 から 2 ピクセルに変化し、パディングは 2 ピクセル減少します。
+            要素がフォーカスを失うと、その境界線は 2 ピクセルから 0 に変化し、パディングは 2 ピクセル増加します。
           </li>
           <li>
-            To ensure the inline SVG arrow graphics in the CSS have sufficient contrast with the background when high contrast settings invert colors, the color of the arrows are synchronized with the color of the text content.
-            For example, the color of the arrow is set to match the foreground color of high contrast mode text by specifying the CSS <code>currentcolor</code> value for the <code>stroke</code> and <code>fill</code> properties of the <code>polygon</code> elements used to draw the arrows.
-            If specific colors were instead used to specify the <code>polygon</code> properties, those colors would remain the same in high contrast mode, which could lead to insufficient contrast between the arrows and the background or even make the arrows invisible if the color matched the high contrast mode background.<br>
-            Note: The SVG element needs to have the CSS <code>forced-color-adjust</code> property set to <code>auto</code> for the <code>currentcolor</code> value to be updated in high contrast mode.
-            Some browsers do not use <code>auto</code> for the default value.
+            CSS のインライン SVG 矢印グラフィックが、ハイコントラスト設定で色が反転した場合に背景と十分なコントラストを持つようにするために、矢印の色はテキストコンテンツの色と同期されます。
+            たとえば、矢印の色は、矢印を描画するために使用される <code>polygon</code> 要素の <code>stroke</code> 及び <code>fill</code> プロパティに CSS <code>currentcolor</code> 値を指定することにより、ハイコントラストモードのテキストの前景色と一致するように設定されます。
+            代わりに特定の色を使用して <code>polygon</code> プロパティを指定した場合、それらの色はハイコントラストモードでも同じままであり、矢印と背景のコントラストが不十分になったり、色がハイコントラストモードの背景と一致した場合は矢印が見えなくなったりする可能性があります。<br>
+            注：SVG 要素は、<code>currentcolor</code> 値をハイコントラストモードで更新するために、CSS <code>forced-color-adjust</code> プロパティを <code>auto</code> に設定する必要があります。
```

Comment:
用語と
受動態を訳出した方がわかりやすそう
```suggestion
            注記：SVG 要素は、<code>currentcolor</code> 値がハイコントラストモードで更新されるようにするために、CSS <code>forced-color-adjust</code> プロパティを <code>auto</code> に設定する必要があります。
```